### PR TITLE
Fix #15 Table output vars 

### LINKED ISSUE #15: Change the variable to gridOutput from textOutput in createTableStructure()

### PR DESCRIPTION
**Pull request by lm-n**
Fix #15 consistent notation throughout the code using "tableOutput"

### CODE REVIEW

File: src/colorNamer.js [hunk 1]
```diff
@@ -117,30 +117,30 @@ var oghsv;
...
-  },
-  {
-    "h":0.0167,
-    "s":0.1176,
-    "b":1,
-    "name":"light pink"
-  },
+	{
+		"h":0,
+		"s":0,
```
Comment (MathuraMG) [inline src/colorNamer.js]: oo same tab thing here

File: examples/Example/app.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+var x =0, speed =2;
```
Comment (MathuraMG) [inline examples/Example/app.js]: can we make this `let` as well?

File: src/colorNamer.js [hunk 3]
```diff
@@ -36,25 +36,25 @@ function calculateColor(hsv) {
...
-      for (var e =0;e<xcp.length;e++){
-        if((xcp[e].h == oghsv[0]) && (xcp[e].s == oghsv[1]) && (xcp[e].b == oghsv[2]) ) {
-            colortext = xcp[e].name;
-            break;
-          }else{
-            colortext = "white";
-          }
-      }
+    	for (var i = oghsv.length - 1; i >= 0; i--) {
+    		oghsv[i]=Math.round(oghsv[i] * 10000) / 10000;
```
[the remark this answers is not in the document]
Comment (lm-n) [reply src/colorNamer.js]: yes that is it. once we choose one we should make the change on colornamer first

File: src/colorNamer.js (hunk 1 above)
Comment (lm-n) [reply src/colorNamer.js]: same thing! lets choose one and I'll change colornamer

Comment (MathuraMG) [reply src/colorNamer.js]: @lm-n 1 tab = 2spaces?

Comment (lm-n) [reply src/colorNamer.js]: sounds good

Comment (MathuraMG) [reply src/colorNamer.js]: I have an atom setup that lints on save if that helps!

**Comment by lm-n** [comment]
@MathuraMG is this ok?

**Comment by lm-n** [comment]
@MathuraMG should I go ahead and merge?

**Comment by MathuraMG** [comment]
👍

**Pull request merged**